core/chain-configs/src/genesis_config.rs

```diff
@@ -293,18 +293,26 @@ impl<'de, F: FnMut(StateRecord)> Visitor<'de> for RecordsProcessor<&'_ mut F> {
     where
         A: MapAccess<'de>,
     {
+        let mut me = Some(self);
```

Comment:
Do we actually need this? It seems to me that we use this to check whether we have already processed `records`, but that could be achieved by checking whether `has_records_field` is true

Reply:
has_records_field is needed to check whether the file has records field at all. Proper config should have exactly on "records" field

Reply:
It's a workaround for rust ownership principles. I know, that "map.next_value_seed" will be called exactly once, but compiler doesn't.